addons/msg-count-badge/addon.json

```diff
@@ -23,6 +23,12 @@
       "id": "showOffline",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Show exact number for large message counts",
```

Comment:
Why make this a setting though if the message count is going to be cut off anyway?

Reply:
If we remove the option, I bet many people will ask the old behavior back for the sake of complaining.

Reply:
I think it's fine, we can consider adding this setting only if people actually complain.

Reply:
1. Secret-chest already complained. 
2. It would be ironic not to have a setting for this, as there is a "Show exact counts" addon. 
3. Some browsers may have higher character limits on extension badges.
4. I don't want to edit this pull request FOUR TIMES adding/removing ONE SETTING. Because I'm sure some reviewers will say "Oh that setting was so cute would you mind adding it back", and the reviewers after them will argue "10 bytes of extension space, screw that useless setting out".

Reply:
**Update:** I now removed the setting as part of [d4125e2](https://github.com/ScratchAddons/ScratchAddons/pull/7317/commits/d4125e22d27764e87d2fbf1248236ef07df725c8). Most of the reviewers agreed on this, and it's not a big thing.